src/autoplex/auto/phonons/flows.py

```diff
@@ -186,9 +189,82 @@ class CompleteDFTvsMLBenchmarkWorkflow(Maker):
     add_dft_rattled_struct: bool = True
     add_rss_struct: bool = False
     displacement_maker: BaseVaspMaker = None
-    phonon_bulk_relax_maker: BaseVaspMaker = None
-    phonon_static_energy_maker: BaseVaspMaker = None
-    rattled_bulk_relax_maker: BaseVaspMaker = None
+    phonon_bulk_relax_maker: BaseVaspMaker | None = field(
+        default_factory=lambda: DoubleRelaxMaker.from_relax_maker(
+            TightRelaxMaker(
+                name="dft tight relax",
+                run_vasp_kwargs={"handlers": {}},
+                input_set_generator=TightRelaxSetGenerator(
+                    user_incar_settings={
+                        # TODO check if user_incar_settings has to be set like that or can be left out
+                        "ALGO": "Normal",
+                        "ISPIN": 1,
+                        "LAECHG": False,
+                        "ISMEAR": 0,
+                        "ENCUT": 700,
+                        "ISYM": 0,
+                        "SIGMA": 0.05,
+                        "LCHARG": False,  # Do not write the CHGCAR file
+                        "LWAVE": False,  # Do not write the WAVECAR file
+                        "LVTOT": False,  # Do not write LOCPOT file
+                        "LORBIT": None,  # No output of projected or partial DOS in EIGENVAL, PROCAR and DOSCAR
+                        "LOPTICS": False,  # No PCDAT file
+                        "NSW": 200,
+                        "NELM": 500,
+                        # to be removed
+                        "NPAR": 4,
+                    }
+                ),
+            )
+        )
+    )
+    phonon_static_energy_maker: BaseVaspMaker | None = None  # field(
+    #     default_factory=lambda: StaticMaker(
+    #         name="dft static",
+    #         input_set_generator=StaticSetGenerator(
+    #             auto_ispin=False,
+    #             user_incar_settings={
+    #                 "ALGO": "Normal",
+    #                 "ISPIN": 1,
+    #                 "LAECHG": False,
+    #                 "ISMEAR": 0,
+    #                 "ENCUT": 700,
+    #                 "SIGMA": 0.05,
+    #                 "LCHARG": False,  # Do not write the CHGCAR file
+    #                 "LWAVE": False,  # Do not write the WAVECAR file
+    #                 "LVTOT": False,  # Do not write LOCPOT file
+    #                 "LORBIT": None,  # No output of projected or partial DOS in EIGENVAL, PROCAR and DOSCAR
+    #                 "LOPTICS": False,  # No PCDAT file
+    #                 # to be removed
+    #                 "NPAR": 4,
+    #             },
+    #         ),
+    #     )
+    # )
```

Comment:
Hey @JaGeo ,
what do you think is better, if we have `None` as default or a StaticMaker as default?